src/main/java/com/puppycrawl/tools/checkstyle/checks/TrailingCommentCheck.java

```diff
@@ -109,7 +109,46 @@
  *   &lt;property name=&quot;format&quot; value=&quot;^\\s*$&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
- *
+ * <p>
```

Comment:
you do not need to modify this check javadoc, please revert changes here

Reply:
> most of them should be added to Examples to show user that this Check is better to use with bunch of satellite Filters.

Sir, in issue it was mention to add specific examples for the user

Reply:
Agree we need test cases.

Does it make more sense we add examples to https://checkstyle.org/config_filters.html#SuppressionXpathFilter or https://checkstyle.org/config_filters.html#SuppressionXpathSingleFilter since these are the areas that deal specifically with xpath suppression?

Reply:
Yup. Shifted examples to SuppressionXpathSingleFilter.